stable/awsebscsiprovisioner/values.yaml

```diff
@@ -33,12 +33,12 @@ provisioner:
   enableVolumeScheduling: false
   image:
     repository: "quay.io/k8scsi/csi-provisioner"
-    tag: "v1.1.0"
+    tag: "v1.2.0"
```

Comment:
Are these tags the versions that have been used in soak and the mini MWT today?

Reply:
That's a good question. I'll check - I got these values from the refactoring PRs diff.

Reply:
According to https://github.com/mesosphere/konvoy-mwt/blob/6ee5b8b186aec7691389fd244ae46aca470db786/konvoymwt2-dr/cluster.yaml#L94 the existing tags were used in the mwt2 (i.e., are set in kubeaddons-configs v0.0.45).

From the comments on https://jira.mesosphere.com/browse/DCOS-56901, it would appear that the MWT susceptible to the same issue.